lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -188,6 +194,13 @@ def create_launch_config(connection, module):
     classic_link_vpc_security_groups = module.params.get('classic_link_vpc_security_groups')
     bdm = BlockDeviceMapping()
 
+    if user_data_path:
+        try:
+            user_data_file = open(user_data_path, 'r')
+            user_data = user_data_file.read()
+        except IOError as e:
+            module.fail_json(msg=str(e))
```

Comment:
Please include the traceback with the failure here, by adding `exception=traceback.format_exc()` to the `fail_json` call.